.github/workflows/release.yaml

```
@@ -289,3 +300,70 @@ jobs:
           release_name: Release ${{ github.event.inputs.release-tag }}
           draft: false
           prerelease: false
+
+  bump-services-version:
+    name: Bump Maven Version
+    if: github.event.inputs.snapshot-build == 'false' && github.event.inputs.services == 'true'
+    runs-on: ubuntu-latest
+    needs:
+      - github-release
+      - release-services
+      - release-services-camunda-ee
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup git
+        run: |
+          git config --global user.email "digiwf@muenchen.de"
+          git config --global user.name "DigiWF Github Bot"
+      - name: Install Java and Maven
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Raise mvn version
+        if: github.event.inputs.snapshot-build == 'false' && github.event.inputs.services-new-version != ''
+        run: |
+          mvn versions:set --batch-mode -DnextSnapshot -DprocessAllModules -DnewVersion=${{ github.event.inputs.services-new-version }}-SNAPSHOT
+          mvn versions:commit
+      - name: Raise mvn version
+        if: github.event.inputs.snapshot-build == 'false' && github.event.inputs.services-new-version == ''
+        run: |
+          mvn build-helper:parse-version versions:set -DnewVersion=\${parsedVersion.majorVersion}.\${parsedVersion.minorVersion}.\${parsedVersion.nextIncrementalVersion}-SNAPSHOT
```

```suggestion
          mvn build-helper:parse-version versions:set -DnewVersion=\${parsedVersion.majorVersion}.\${parsedVersion.nextMinorVersion}.0-SNAPSHOT
```

Das hat sich geändert. Wir wollen die Minor nun nutzen für neue Releases und BugFix-Versions wirklich nur für Hotfixes